### PR TITLE
1 getting started readme

### PR DESCRIPTION
**Pull request by davidmikh**

### CODE REVIEW

File: content/efcore/getting-started/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Getting Started using Entity Framework Core
+by [The Raikes School](http://cse.unl.edu/raikes-school)
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:2]: Update from Maria: Leave out the author(s) for now. We can add them later.

File: content/efcore/getting-started/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## What is Entity Framework Core?
+
+Entity Framework Core (EF Core) is a lightweight, extensible, and cross-platform version of Entity Framework. Entity Framework is an object-relational mapper (O/RM) that enables .NET developers to work with a database using .NET objects. This allows developers to create and manage databases without most of the data-access code that developers usually need to write.�
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:6]: image

Unrecognized character at end of line.

File: content/efcore/getting-started/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Who is this tutorial meant for?
+
+This tutorial is meant for those new or unfamiliar to EF Core, but if you already have an understanding of EF Core, consider this a refresher course. This tutorial will also help you navigate EF Core if you are familiar with previous versions of EF. For those more experienced EF Core programmers, [the docs](https://docs.efproject.net/en/latest/) may be of more use to you. The docs feature higher level EF Core concepts that this tutorial does not.
+
+If you know what you're looking for, feel free to browse by topic on the side. If not, welcome! Let's get started together!
+
+## Prerequisites for this tutorial
+
+This tutorial assumes a knowledge of other programming concepts, including:�
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:18]: It sounds weird to say "This tutorial assumes a knowledge of other programming concepts, including: basic programming concepts". 

How about "This tutorial assumes knowledge of the following topics:"

File: content/efcore/getting-started/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Getting Started using Entity Framework Core
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:1]: ~~using~~ with

File: content/efcore/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Entity Framework Core Tutorials
+
+[Getting Started with Entity Framework Core](getting-started/README.md)
+This tutorial covers the basics of how to get started writing your first application using Entity Framework Core.
```
Comment (helmoski) [inline content/efcore/README.md:4]: This tutorial covers the basics of using Entity Framework Core in a C# application.

File: content/efcore/getting-started/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+This tutorial assumes a knowledge of other programming concepts, including:�
+
+* Basic programming concepts�
+* C# (see the [interactive C# tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started))
+* Relational database concepts
+	- SQL-type databases will be the focus of this tutorial, though other databases function with EF Core
+* LINQ statements (see this [interactive tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) for an introduction to LINQ)�
+
+If you're a new to these topics, you can check out their tutorials. If you're only a little rusty, we'll remind you of the relevant aspects along the way.
```
Comment (rdahlman3) [inline content/efcore/getting-started/README.md:26]: Typo - remove the `a` in `If you're a new to these topics`

File: content/efcore/getting-started/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# Getting Started using Entity Framework Core
+by [The Raikes School](http://cse.unl.edu/raikes-school)
+
+## What is Entity Framework Core?
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:4]: To better match the C# tutorial, let's add a chapter called "Introducing EF Core" and put this "What is Entity Framework Core?" content there. Then `getting-started/README.md` will only have the target audience, prereqs, and a table of contents. (see [C# tutorial README](https://github.com/LadyNaggaga/training-tutorials/blob/master/content/csharp/getting-started/README.md))

File: content/efcore/getting-started/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+As a lightweight version of the original Entity Framework, EF Core does not have all the features available in the latest version of Entity Framework (EF6.x). However, EF Core does offer a few features EF6.x does not. For more on these differences, check out [the docs](https://docs.efproject.net/en/latest/efcore-vs-ef6/features.html). If you have further questions about which version is best for your project, visit ["Which One Is Right for You?"](https://docs.efproject.net/en/latest/efcore-vs-ef6/choosing.html).
+
+## Who is this tutorial meant for?
+
+This tutorial is meant for those new or unfamiliar to EF Core, but if you already have an understanding of EF Core, consider this a refresher course. This tutorial will also help you navigate EF Core if you are familiar with previous versions of EF. For those more experienced EF Core programmers, [the docs](https://docs.efproject.net/en/latest/) may be of more use to you. The docs feature higher level EF Core concepts that this tutorial does not.
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:12]:
``` markdown
This tutorial is meant as an introduction for those new to Entity Framework Core. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/). You should understand the following concepts before before starting this tutorial:
* C# (see the [interactive C# tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started))
* Relational Databases
```

File: content/efcore/getting-started/README.md (hunk 6 above)
Comment (helmoski) [inline content/efcore/getting-started/README.md:26]: Remove

File: content/efcore/getting-started/README.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+* Basic programming concepts�
+* C# (see the [interactive C# tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started))
+* Relational database concepts
+	- SQL-type databases will be the focus of this tutorial, though other databases function with EF Core
+* LINQ statements (see this [interactive tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) for an introduction to LINQ)�
+
+If you're a new to these topics, you can check out their tutorials. If you're only a little rusty, we'll remind you of the relevant aspects along the way.
+
+Now, let's get started!
```
Comment (helmoski) [inline content/efcore/getting-started/README.md:28]: Remove

**Review by helmoski — CHANGES_REQUESTED** [review]
- [ ] Remove Author(s)
- [ ] Reorganize to match C# tutorial
  - [ ] Introducing EF Core chapter
  - [ ] Add table of contents to `content/efcore/getting-started/README.md`
- [ ] Some rewording and cutting out of content
- [ ] Remove unrecognized characters

**Comment by helmoski** [comment]
We decided to make significant reorganization changes, so we we will need to adjust branches and issues accordingly. Please submit a new pull request with a branch that reflects these changes.

**Comment by arynhuck** [comment]
Working on this now. Commenting as stuff is amended.

File: content/efcore/getting-started/README.md (hunk 1 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:2]: Author removed

File: content/efcore/getting-started/README.md (hunk 4 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:1]: Fixed

File: content/efcore/getting-started/README.md (hunk 7 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:4]: What is Entity Framework Core? has been removed. Consider renaming this section.

File: content/efcore/getting-started/README.md (hunk 3 above)
Comment (arynhuck) [inline content/efcore/getting-started/README.md:18]: Fixed

Comment (arynhuck) [reply content/efcore/getting-started/README.md:18]: Fixed.

File: content/efcore/getting-started/README.md (hunk 8 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:12]: Keeping LINQ, but I used your suggested edit to cut down on the wordage.

File: content/efcore/getting-started/README.md (hunk 6 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:26]: Cut the database jibberish. Left LINQ.

File: content/efcore/getting-started/README.md (hunk 9 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md:28]: Done.

File: content/efcore/README.md (hunk 5 above)
Comment (arynhuck) [reply content/efcore/README.md:4]: Fixed.

**Pull request closed without merging**